data/multiplayer/README.multiplayer

```diff
@@ -93,63 +92,68 @@ As for the remaining great wonders:
   anarchy-from-disorder effects of Democracy to all nations,
   regardless of government.
 
-Building changes
-----------------
+Building Changes
+^^^^^^^^^^^^^^^^
 
 - Colosseum: Renamed to "Amphitheater", and cost decreased to 60 (was 70).
 - Supermarket: Increases farmland tile food output by 100% (was 50%).
 - Police Station: Affects 2 and 4 unhappy citizens under Republic and
   Democracy respectively (was 1 and 2).
 
-Government changes
-------------------
+Government Changes
+^^^^^^^^^^^^^^^^^^
 
 The Fundamentalism government is added. This is similar to the one in
 the civ2 ruleset, with following exceptions:
 - Food upkeep for Settlers is 3
 - Free unit upkeep is worth 10 shields in each city
 
-Technology changes
-------------------
+Technology Changes
+^^^^^^^^^^^^^^^^^^
 
 - Philosophy gives a free tech to each player when they research it
   (not just the first player).
 - Discovery of Environmentalism additionally knocks 50% off cities'
   pollution from production.
 - Added Fundamentalism from the civ2 ruleset.
 
-Unit changes
-------------
+Unit Changes
+^^^^^^^^^^^^
+
 - Unit movement rates are twice the values in the classic ruleset.
 - Caravans and Freight cannot create trade routes.
 - Caravans cost 30 production (was 50); Freight remain at 50.
 - Elephants, Crusaders, and Fanatics added from the civ2 ruleset.
 
-Default settings
-----------------
+Default Settings
+^^^^^^^^^^^^^^^^
 
 The ruleset suggests some server setting changes, although these can
 be overridden. Notably, huts, barbarians, national borders, and
 diplomacy are all disabled, more initial units are given, and "fairer"
 map settings are used.
 
 
-2. Use the new ruleset
-   -------------------
+Use the New Ruleset
+===================
 
 To play a game with the multiplayer rules, start the server with:
-       ./fcser -r data/multiplayer.serv
+
+$ ./freevic21-server -r data/multiplayer.serv
+
 or a single player game with
-       ./freeciv-gtk3 -r data/multiplayer.serv
+
+$ ./freeciv21-client -r data/multiplayer.serv
```

Comment:
Same question here

Reply:
same answer. I didn't want to add a bunch of rst type code to the plain text file so tried to add simple stuff that would at least look like plain text but still render ok in RTD.